css/style.css

```diff
@@ -0,0 +1,24 @@
+#main{
+    text-align: center;
+    font-size: 1.6rem;
+    line-height: 1.5;
+    margin: 0;
+    background-color: #FEFBE9;
+    
+}
+#list-items{
+    display: inline-block;
+    text-align: left;
+}
```

Comment:
Please, try adding some margin to this list, so it will be visually more readable!😉